BookPlayer/Library/FileManagement/ImportOperation.swift

```diff
@@ -21,15 +22,67 @@ class ImportOperation: Operation {
         self.files = files
     }
 
+    func handleZip(file: FileItem) {
+        // Unzip to a temp directory
+        if file.originalUrl.pathExtension == "zip" {
+            let tempURL = file.destinationFolder.appendingPathComponent("tmp")
+
+            do {
+                try FileManager.default.createDirectory(at: tempURL, withIntermediateDirectories: true, attributes: nil)
+                try FileManager.default.unzipItem(at: file.originalUrl, to: tempURL)
+
+            } catch {
+                print("Extraction of ZIP archive failed with error:\(error)")
+            }
+
+            do {
```

Comment:
This `do` doesn't have a `catch`, and everything else is already in a inner `do`, we can safely delete this